packages/perspective-viewer-d3fc/src/js/gridlines/gridlines.js

```diff
@@ -46,9 +70,37 @@ const withGridLines = (seriesMulti, annotationGridline, mainGrid, crossGrid) =>
         return _withGridLines;
     };
 
-    return _withGridLines;
-};
+    _withGridLines.mode = (...args) => {
```

Comment:
Apologies for not mentioning this sooner, but the `zoomableChart` component has a similarly behaving property, but it is called `canvas` (true/false), rather than `mode`.

Ideally both properties would be consistent.

As we don't envision any further alternatives and canvas is the exception to the rule, I think have it as a boolean is sufficient, but I'm not particularly attached to it.